phpBB/viewtopic.php

```diff
@@ -142,9 +140,10 @@
 		}
 		else
 		{
+			$forum_id = $row['forum_id'];
 			$sql = 'SELECT topic_id, forum_id
 				FROM ' . TOPICS_TABLE . '
-				WHERE forum_id = ' . $row['forum_id'] . "
+				WHERE forum_id = ' . $forum_id . "
```

Comment:
type casting:

`WHERE forum_id = ' . (int) $forum_id . '`

same applies to any occurance here.

Reply:
Just for clarity, are you asking for all cases of (int) php variables in sql expressions to be cast?  Including those not otherwise touched by this PR?

Reply:
That's up to you, at least yours.